src/handlers/issue_comment/created.js

```diff
@@ -40,8 +40,20 @@ const newIssueBody = (user: string, originalUri: string, body: string) => {
     return `Original issue submitted by @${user} in ${originalUri}\n\n${body}`;
 };
 
+const isAnnoying = (body) => body === '+1' || body === '-1';
+
 // This is super messy and needs cleanup. But not tonight :D
 export default function({ comment, issue, repository }: IssueCommentPayload) {
+
+    if (isAnnoying(comment.body.trim())) {
+        log(`Removing a +1/-1 comment`, 'verbose');
+        github.deleteIssueComment({
+            id: comment.id,
+            owner: repository.owner.login,
+            repo: repository.name
+        });
```

Comment:
Since we can't do a reaction for them

        // .then(() => {
        //     github.createIssueReaction({
        //         number: issue.number,
        //         owner: repository.owner.login,
        //         repo: repository.name,
        //         content: comment.body
        //     });
        // });

Reply:
just to understand this, it deletes the comment, but it does not add a reaction (although its shown in the gif)?

Reply:
Yeah the issue I realized is that it would create a reaction from the bot not the person because we don't have access